charts/flagger/templates/pdb.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.podDisruptionBudget }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ template "flagger.name" . }}
+spec:
+  minAvailable: {{ .Values.podDisruptionBudget.minAvailable }}
```

Comment:
Please add this to values.yaml and document it in the chart readme

Reply:
Added default values to `values.yaml` and added two entries to the chart README.

Selected to go with not enabled as default as the default is to not use leader election
with only one replica.